langchain_opea/integrations/pipeline.py

```diff
@@ -0,0 +1,83 @@
+import os
+import sys
+
+import torch
+from transformers import TextGenerationPipeline
+
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(SCRIPT_DIR))
+
+
+class GaudiTextGenerationPipeline(TextGenerationPipeline):
```

Comment:
suggest renaming it to _GaudiTextGenerationPipeline as it's just an internal class for use.